clients/samples/example_hipblaslt_preference.cpp

```diff
@@ -64,9 +64,9 @@
 #define DIM2 1024
 #define DIM3 1024
 #define BATCH_COUNT 1
-#define ALPHA 2
-#define BETA 3
-#define BENCH_LOOP_COUNT 3
+#define ALPHA 1
```

Comment:
alpha and beta can be set by input command, for default, please use alpha=2 and beta=3